dev-tools/run.sh

```diff
@@ -115,6 +110,7 @@ else
     fi
 
     # Start Integreat CMS
+    pipenv run integreat-cms-cli tailwind start &\
```

Comment:
```suggestion
    sudo -u $SUDO_USER env PATH="$PATH" pipenv run integreat-cms-cli tailwind start &
```
In this part of the script, all commands are executed as sudo to make sure the docker commands work even for users who aren't in the docker group. This means that for all commands not requiring sudo, we have to de-escalate privileges to the user who ran `sudo` in the first place (which is automatically stored in the env var `$SUDO_USER`).